### PR TITLE
Fixed branch alias

### PR DESCRIPTION
**Pull request by GrahamCampbell**

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -22,7 +22,7 @@
     },
     "extra": {
         "branch-alias": {
-            "dev-master": "4.9-dev"
+            "dev-master": "4.10-dev"
```
Comment (nikic) [inline composer.json:25]: Is it possible to make this `4.x-dev`?

Comment (GrahamCampbell) [reply composer.json:25]: Pinning the minor version is better so that composer knows the difference between `~4.9.0@dev`, `~4.10.0@dev` , `~4.11.0@dev`.

Comment (GrahamCampbell) [reply composer.json:25]: Alternatively, naming the branch `4.10` avoids needing to use and update the aliases at all.

Comment (nikic) [reply composer.json:25]: I'm happy to remove the branch-alias altogether if 4.x-dev is not acceptable.

**Pull request closed without merging**